pkg/backup/client.go

```diff
@@ -176,14 +193,20 @@ func (bc *Client) SetStorage(ctx context.Context, backend *backuppb.StorageBacke
 		return errors.Annotatef(err, "error occurred when checking %s file", metautil.MetaFile)
 	}
 	if exist {
-		return errors.Annotate(berrors.ErrInvalidArgument, "backup meta exists, may be some backup files in the path already")
+		errMsg := fmt.Sprintf(`backup meta file exists in %v, 
+							there may be some backup files in the path already, 
+							please specify a correct backup directory!`, GetLockOrMetaFilePath(metautil.MetaFile, backend))
```

Comment:
`GetLockOrMetaFilePath(metautil.MetaFile, backend)` should be equivalent to `bc.storage.URI() + "/" + metautil.MetaFile`.

Reply:
removed redundant code